src/client/components/Pokedex.jsx

```diff
@@ -1,7 +1,104 @@
+import { useState, useEffect } from "react";
+
+import "../styling/pokedex.css"
+import ratingStar from "../assets/star.png"
+import ratingStarHollow from "../assets/Star_Outline_2.png"
+
+const URL = process.env.REACT_APP_API_URL;
+const pokemonEndpoint = "/pokemon";
```

Comment:
No "Magic Strings". Nice :)

Reply:
but isn't this a magic string? `const pokemonEndpoint = "/pokemon";`

Reply:
but it's in a constant right?

Reply:
I suppose - and indeed, that's better, but I'd prefer all my string constants in some string constants file that I then import where necessary. That way, I know where to find them all.